sunpy/io/tests/test_jp2.py

```diff
@@ -30,3 +32,10 @@ def test_read_memmap():
     # Keyword is not passed in the function call
     data, _ = read(AIA_193_JP2, memmap=False)[0]
     assert data.base is not None
+
+def test_simple_write(tmpdir):
+    data, header = _fits.read(TEST_AIA_IMAGE)[0]
+    outfile = tmpdir / "test.jp2"
+    jp2.write(str(outfile), data, header)
+    assert outfile.exists()
+
```

Comment:
Would a data or header == the fits one (after conversion) be possible?

Reply:
not for data, the uint8 conversion for glymur changes what the data array looks like.

For header it seems like it would be possible, but not as trivial as it sounds because the "read" functions seem to read differently, which may be an error on its own. It seems like _fits.read creates a header out of strings and jp2.read creates a header out of real values. I don't know where the casts happen between reading/writing.

I tried to compare the headers and found that
- _fits.read header will put in `'nan'` but jp2.read will use `nan` (str vs. real nan)
- _fits.read header has `'001.197'` and jp2.read gives `1.197`

```python
FAILED test_jp2.py::test_simple_write - AssertionError: assert '001.197' == 1.197
```

Reply:
Added a read back and sanity check on the header, let me know if you think that's sufficient

Reply:
It works for me!

Reply:
Have we documented the fact that the data is modified by saving it to jp2?

Reply:
It is done now.